test/solidity/Periphery/ReceiverChainflip.t.sol

```diff
@@ -391,7 +391,7 @@ contract ReceiverChainflipTest is TestBase {
         swapData[0] = LibSwap.SwapData({
             callTo: address(uniswap),
             approveTo: address(uniswap),
-            sendingAssetId: LibAsset.NATIVE_ASSETID,
+            sendingAssetId: LibAsset.NULL_ADDRESS,
             receivingAssetId: ADDRESS_DAI,
             fromAmount: amountIn,
             callData: abi.encodeWithSelector(
```

Comment:
I see GasZipFacet also has it's own definition of NON_EVM_ADDRESS

Reply:
This has been updated